source/nodejs/adaptivecards-designer/src/containers/viva-connections/viva-connections-container-light.css

```diff
@@ -25,9 +25,9 @@
   padding: 5px 8px;
   bottom: 12px;
   overflow: hidden;
-  border-color: #696baa;
+  border-color: #7579EB;
   background-color: #ffffff;
-  color: #6264a7;
```

Comment:
Same.
Why are we making changes in the light.css file??

Reply:
See other reply!